src/magento/security-action-plan.md

```diff
@@ -5,22 +5,22 @@ group: operations
 
 If you suspect that your site is compromised, follow this action plan without delay.
 
-1. **DIAGNOSE**
+1. **Diagnose**
 
     Run a scan to establish the security status of your Magento store. [MageReport.com](https://www.magereport.com/) is a highly regarded service that is available at no charge to members of the Magento community.
 
-1. **CLEAN**
+1. **Clean**
 
     Hire a [qualified consultant](https://magento.com/partners/portal/directory?partner_type=1) or online service to clean your site of all malicious code. Some Magento community members recommend [Sucuri Website Malware Removal](https://sucuri.net/website-antivirus/malware-removal). Check the `/media` folder for leftover executable code. Remove all unknown Admin users and reset all Admin passwords.
 
-1. **PROTECT**
+1. **Protect**
 
-    Keep your Magento installation up-to-date with the most current release. If using an older version, apply all security patches as they become available. Review and follow [Magento Security Best Practices](https://magento.com/security/best-practices/security-best-practices). Subscribe to [Magento Security Alerts](https://magento.com/security/sign-up).
+    Keep your Magento installation up-to-date with the most current release. If you are using an older version, apply all security patches as they become available. Review and follow [Magento Security Best Practices](https://magento.com/security/best-practices/security-best-practices). Subscribe to [Magento Security Alerts](https://magento.com/security/sign-up).
```

Comment:
i think hyphens are only needed when used as an adjective.   i don't think up to date needs -s here.